execute/wait_test.go

```diff
@@ -42,7 +42,7 @@ func (h *testHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		time.Sleep(h.delay)
 	}
 	if h.notReadyCount > 0 {
-		h.notReadyCount -= 1
+		h.notReadyCount--
```

Comment:
I find -= 1 so much more readable. Its a matter of taste, and golint doesn't agree with me obviously ;)